torch/jit/annotations.py

```diff
@@ -246,6 +246,8 @@ def try_ann_to_type(ann, loc):
         return TensorType.get()
     if ann is torch.Tensor:
         return TensorType.get()
+    if inspect.isclass(ann) and issubclass(ann, torch.Tensor):
```

Comment:
I think this also covers the case above, so we can delete it.

Reply:
Yep, deleted. Thanks for catching this.